cmd/editorconfig-checker/main.go

```diff
@@ -64,6 +65,10 @@ func init() {
 
 	currentConfig, _ = config.NewConfig(configPaths)
 
+	if strings.HasSuffix(currentConfig.Path, ".ecrc") {
```

Comment:
I ask everyone to consider doing this

```suggestion
	if configFilePath == "" && strings.HasSuffix(currentConfig.Path, ".ecrc") {
```

This way we won't report a problem to user when they provide a path with .ecrc with --config

The warning added via this feature is about asking the use to rename the config file. For me we should limit ourselves to when no --config is provided. Warning them in such case would be a problem.

Reply:
Hm.. we could declare this a conscious decision - after all that is to encourage change to the new name. And some tooling like super-linter will automatically add a `--config` when they find a matching find in their directory (in this example `.github/linters/`

Reply:
> encourage change to the new name

So if we take the decision that it's on purpose, that we also warn if the user provide `--config .ecrc`, to encourage using another name, are your (@ccoVeille) reviews blocking? Is this PR ready to be merged as is?

Reply:
Non blocking.